Makefile

```diff
@@ -100,6 +102,18 @@ image/build/push: image/build image/push
 image/build/test:
 	operator-sdk build --enable-tests $(INTEGREATLY_OPERATOR_IMAGE)
 
+.PHONY: image/osd/build
+image/osd/build:
+	go mod vendor
+	docker build . -f Dockerfile.osd.addon -t $(INTEGREATLY_OPERATOR_TEST_HARNESS_IMAGE)
+
+.PHONY: image/osd/push
+image/osd/push:
+	docker push $(INTEGREATLY_OPERATOR_TEST_HARNESS_IMAGE)
+
+.PHONY: image/osd/build/push
+image/osd/build/push: image/osd/build image/osd/push
+
```

Comment:
Could we consider adding a new make file for these targets? https://github.com/integr8ly/integreatly-operator/tree/master/make. make/osd.mk ?

Reply:
sure